### PR TITLE
build for linux

### PR DESCRIPTION
**Pull request by atinylittleshell**
I'm playing wow on ubuntu these days. Adding a build for linux here

### CODE REVIEW

File: packages/app/src/nativeBridge/modules/mainWindowModule.ts [hunk 1]
```diff
@@ -37,7 +38,11 @@ export class MainWindowModule extends NativeBridgeModule {
 
   @moduleFunction({ isRequired: true })
   public async hideToSystemTray(mainWindow: BrowserWindow): Promise<void> {
-    mainWindow.hide();
+    if (platform === 'win32') {
+      mainWindow.hide();
```
Comment (prsauer) [inline packages/app/src/nativeBridge/modules/mainWindowModule.ts:42]: presumably the app.quit call is the behavior we want on macOS as well? I know they have a tray but it's not quite the same as win

**Comment by prsauer** [comment]
Is the on `second-instance` behavior still okay? I remember seeing some notes reviewing docs for deep linking that some of that behavior could be a bit different

**Comment by prsauer** [comment]
> Is the on `second-instance` behavior still okay? I remember seeing some notes reviewing docs for deep linking that some of that behavior could be a bit different

https://www.electronjs.org/docs/latest/tutorial/launch-app-from-url-in-another-app
Was reviewing this on a different project as I was testing oauth using the user's browser (instead of an electron window) and a callback with an app url (like wowarenalogs://someauthCallback)

File: packages/app/src/nativeBridge/modules/mainWindowModule.ts (hunk 1 above)
Comment (atinylittleshell) [reply packages/app/src/nativeBridge/modules/mainWindowModule.ts:42]: lol i've completely forgotten how it worked on mac and i don't have one now... probably safe to follow the quit behavior though

**Comment by atinylittleshell** [comment]
> Is the on `second-instance` behavior still okay? I remember seeing some notes reviewing docs for deep linking that some of that behavior could be a bit different

Yes - that won't be a problem. The second-instance behavior is built such that if a new process starts while an existing process of WAL is already running, the new process will kill itself, and the existing process will simply show the main window if that's hidden. 

This logic shouldn't interfere with it, as this simply makes the existing process exit more often. Second instance will then become the main process and launch as usual.

**Review by prsauer — APPROVED** [review]

**Pull request merged**